script/youdao_translator_api.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not sure if this code is wrote by you.
Could you add a license header for this file if you wrote it ?